homeassistant/components/hue/light.py

```diff
@@ -443,4 +449,5 @@ def device_state_attributes(self):
         attributes = {}
         if self.is_group:
             attributes[ATTR_IS_HUE_GROUP] = self.is_group
+        attributes[ATTR_REACHABLE] = self.reachable
```

Comment:
We shouldn't add this. This attribute does not what you think it does. It means the hub can't reach the light for whatever reason (cut power, anything else).

Reply:
HI Balloob,

I exactly want it to do just  that. We need to make a difference between unavailable (like what happens now the the hub loses connection to the light) and correct normal operation with a light that is off-power.

Iow: 
unavailable signals an (Hub) issue,
reachable shows a regular attribute with valuable information of the current state.

I do think the logic of the def available(self) should be adjusted if this attribute is added, so make the difference clear.